### DISCUSSION TITLE
Download Link not working

### DISCUSSION THREAD

**Discussion by SegAgeS** [trakt-tools/universal-trakt-scrobbler#173] [General]
Is there an alternative download link to get this?  It seems like the link given is not valid any longer.
Thanks

**Comment by lolo020241** [comment]
the same with me, 404 error code

**Comment by Orinks** [comment]
You can just go to the tags under beta and get the zip that way. It doesn't work though. I can download the source with Github desktop, which was updated in June, so most recent, but not sure what to build it with.
Chrome or Edge fail to read the beta version Manifest.

**Comment by Brandon1212b** [comment]
Having the same problem. Is this project still on going? How do we install

Edit: Was able to find the download. On the right under releases click 1 tag, then click beta
